gnuradio-runtime/lib/tagged_stream_block.cc

```diff
@@ -44,14 +47,74 @@ void tagged_stream_block::forecast(int noutput_items,
     }
 }
 
+void tagged_stream_block::get_tags_in_range(std::vector<tag_t>& v,
```

Comment:
Does this not produce compiler warnings?

Reply:
nope, this is polymorphism, non-dynamic.